integration/integration_test.py

```diff
@@ -908,19 +927,21 @@ def _temp_path():
         generate_decimal_case(),
         generate_datetime_case(),
         generate_nested_case(),
-        generate_dictionary_case()
+        generate_dictionary_case().skip_category(SKIP_FLIGHT),
     ]
 
     if flight:
-        file_objs.append(generate_primitive_case([32 * 1024],
+        file_objs.append(generate_primitive_case([24 * 1024],
                                                  name='large_batch'))
 
     generated_paths = []
     for file_obj in file_objs:
         out_path = os.path.join(tempdir, 'generated_' +
                                 file_obj.name + '.json')
         file_obj.write(out_path)
-        generated_paths.append(out_path)
+        generated_paths.append(TestCase(name=file_obj.name,
```

Comment:
Ok, sorry to come back to this, but why do you need both `JsonFile` and `TestCase`? They seem to represent the same thing...

Reply:
No worries. It's because of `get_static_json_files` below. We could read the existing JSON into Python and put it into a JsonFile, but that seems like a waste. Or, I suppose, JsonFile could handle both cases (generated and static data) - I can make that change.

Reply:
I went ahead and merged the two.